crates/store/re_types/definitions/rerun/datatypes/pixel_format.fbs

```diff
@@ -24,13 +28,82 @@ enum PixelFormat: ubyte {
     // this organization and subsequently reduce the chance we may find ourselves wanting to
     // change the values in the future.
 
-    /// `NV12` (aka `Y_UV12`) is a YUV 4:2:0 chroma downsampled format with 12 bits per pixel and 8 bits per channel.
+    /// `Y_U_V24` is a YUV 4:4:4 fully planar YUV format without chroma downsampling, also known as `I444`.
+    ///
+    /// This uses limited range YUV, i.e. Y is expected to be within [16, 235] and U/V within [16, 240].
+    ///
+    /// First comes entire image in Y in one plane, followed by the U and V planes.
+    Y_U_V24_LimitedRange = 39,
```

Comment:
the ordering in respect to the numbers in here is rather dubious
I tried to order this logically rather than by enum numbers (which are derived from Ocean). But that makes it hard for the codes beyond Ocean :/